src/swapd/runtime.rs

```diff
@@ -1460,6 +1461,25 @@ impl Runtime {
                 self.pending_peer_request.push(msg);
             }
 
+            CtlMsg::SetAddressSecretKey(address_secret_key) => {
+                if let AddressSecretKey::Monero {
+                    address,
+                    mut secret_key_info,
+                } = address_secret_key
+                {
+                    secret_key_info.creation_height = self.monero_address_creation_height;
+                    endpoints.send_to(
+                        ServiceBus::Ctl,
+                        ServiceId::Wallet,
```

Comment:
Are we forced to impersonate wallet here? For me it seems a bad practice if we are not forced to.

Reply:
No longer relevant, since the wallet logic now lives in the swap daemon.